server/controllers/userController.js

```diff
@@ -0,0 +1,157 @@
+const User = require('../models').User,
```

Comment:
constants should be declared on separate lines. consider the right format for importing modules  if your project uses es6.